xbmc/filesystem/IFile.h

```diff
@@ -39,6 +39,14 @@ class CURL;
 namespace XFILE
 {
 
+enum SpecialReadSizeRequest
```

Comment:
Can we rename this to something different? Perhaps something like ReadSizeRequestCode ?

Reply:
Sure!